4-WebApp-your-API/4-1-MyOrg/README-use-certificate.md

```diff
@@ -0,0 +1,93 @@
+# Web app using client certificates with Microsoft.Identity.Web
+
+There are different approaches to [use certificates with Microsoft.Identity.Web](https://github.com/AzureAD/microsoft-identity-web/wiki/Certificates). In this Readme, we will explore use of certificate from a path on the disk.
+
+## Use Self Signed Certificate from a path
+
+To use certificates instead of an application secret you will need to do little changes to what you have done so far:
+
+- generate a certificate and export it, if you don't have one already
+- register the certificate with your application in the Azure AD portal
+- Update appsettings.json
+
+### Generate a certificate
+
+#### On Windows
```

Comment:
Instead of "On Windows" vs "On other OSs", it might be better to say "with Powershell" vs. "with OpenSSL"